### DISCUSSION TITLE
Add support for setting all tools as global default

### DISCUSSION THREAD

**Discussion by zhiqiangxu** [jdx/mise#4003] [Q&A]
When `mise.toml` contains a lot of tools, it's handy to have a command to set all tools as global default.

**Comment by jdx** [comment]
I don't understand what this means

**Comment by zhiqiangxu** [comment]
> I don't understand what this means

I want to access all tools specified in [`mise.toml`](https://github.com/ethereum-optimism/optimism/blob/develop/mise.toml) while in another directory(not the directory containing `mise.toml`). 

I don't want to do `mise use --global xxx@version` one by one.

So I think it's handy to have such a command, WDYT?

**Comment by jdx** [comment]
`cp mise.toml ~/.config/mise/conf.d/extras.toml`

**Comment by zhiqiangxu** [comment]
> cp mise.toml ~/.config/mise/conf.d/extras.toml

It works! Oh, it's documented [here](https://mise.jdx.dev/configuration.html).

Thanks!

**Comment by zhiqiangxu** [comment]
> `cp mise.toml ~/.config/mise/conf.d/extras.toml`

This works for stand-alone terminal, but vs code can't find the installed binaries:

<img width="437" alt="image" src="https://github.com/user-attachments/assets/a821f8ed-5c99-42bf-ab75-5c8f803547bd" />

**Comment by zhiqiangxu** [comment]
Fixed by installing the [mise-vscode](https://github.com/hverlin/mise-vscode) plugin.

**Comment by zhiqiangxu** [comment]
> `cp mise.toml ~/.config/mise/conf.d/extras.toml`

Hey @jdx , is it possible to temporarily disable mise for some specific bash command after this?

In my case, `DOCKER_REPO=blockscout-optimism docker compose -f geth.yml up` no longer works after `cp mise.toml ~/.config/mise/conf.d/extras.toml`, so I want to de-activate it(but for this invocation only).
